sdk/src/utils/graphs/adjacency-pool-graph.ts

```diff
@@ -0,0 +1,195 @@
+import { AddressUtil } from "@orca-so/common-sdk";
+import { Address } from "@project-serum/anchor";
+import { PublicKey } from "@solana/web3.js";
+import {
+  Hop,
+  PoolGraph,
+  PoolGraphUtils,
+  PoolTokenPair,
+  Route,
+  RouteFindOptions,
+  RouteSearchEntires
+} from "../public/pool-graph";
+
+export class AdjacencyPoolGraph implements PoolGraph {
+  readonly graph: {
+    [k: string]: PoolGraphEdge[];
+  };
+
+  constructor(pools: PoolTokenPair[]) {
+    this.graph = buildPoolGraph(pools);
+  }
+
+  getRoute(startMint: Address, endMint: Address, options?: RouteFindOptions): Route[] {
+    const [startMintKey, endMintKey] = [
```

Comment:
nit: this method can just be `getAllRoutes([[startMint, endMint]], options)`

Reply:
yikes. good call.